pkg/core/zap.go

```diff
@@ -0,0 +1,118 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package core
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/dubbogo/pixiu-admin/pkg/global"
+	"github.com/dubbogo/pixiu-admin/pkg/utils"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+var level zapcore.Level
+
+func Zap() (logger *zap.Logger) {
+	if ok, _ := utils.PathExists(global.CONFIG.Zap.Director); !ok {
+		fmt.Printf("create %v directory\n", global.CONFIG.Zap.Director)
```

Comment:
only one return value not need to be quoted

Reply:
![image](https://user-images.githubusercontent.com/18587688/142858085-589e090c-0452-44be-b8bd-e18da0aeef41.png)
zap.New what is returned is a pointer

Reply:
variable reference